### ISSUE TITLE
Questions about negative loss in paper.

### ISSUE THREAD

**Issue by mls1999725** [IrvingMeng/MagFace#4]
Hi, I'm very interested in your work. I wonder what is negative loss in Figure 3 of MagFace paper? Is that the value of cross-entropy loss after softmax?

**Comment by IrvingMeng** [comment]
We appreciate your interests in MagFace.
The negative loss seems ambiguous and we will clarify it more clearly in the next version. It is actually the negative value of the cross entropy loss, i.e., the log(softmax) value. Easy samples have large softmax values and so are log(softmax) values.

**Comment by mls1999725** [comment]
I get it, Thanks a lot!
